src/SparkFun_AS7331.h

```diff
@@ -47,10 +47,17 @@ class SfeAS7331ArdI2C : public SfeAS7331Driver
 
         setCommunicationBus(&_theI2CBus);
 
+        // If the address passed in isn't the default, set the new address.
+        if (kDefaultAS7331Addr != address)
+        {
+            if(!setDeviceAddress(address))
+                return false;
+        }
+
```

Comment:
This is not needed, because it's handled already by calling `_theI2CBus.init(wirePort, address)` a few lines above.

Reply:
Good point.